tools/bash-completion

```diff
@@ -1,38 +1,42 @@
 # list of commands/arguments
```

Comment:
is it possible to automatically generate this file? @liuzhe-lz

Reply:
Maybe we can just write the whole file's content to the correct location.